coda-prototype/src/ui/editorUI.js

```diff
@@ -254,11 +254,7 @@ var codeEditorManager = {
                     .removeClass("has-error");
 
                 newDataset.schemes[newId] = tempScheme;
-                messageViewerManager.codeSchemeOrder = [newId + ""].concat(messageViewerManager.codeSchemeOrder);
-                activeSchemeId = newId + "";
-                messageViewerManager.activeScheme = newId + "";
-
-                messageViewerManager.addNewSchemeColumn(tempScheme, name);
+                messageViewerManager.addNewSchemeColumn(tempScheme);
```

Comment:
Is there a lint check we can run somewhere to detect these?

Reply:
There's one built into WebStorm, but it's generating a LOT of errors. It would probably be more practical to examine these when migrating each individual file to TS, which would pick up an error like this.